subsys/bluetooth/controller/ll_sw/ull_adv.c

```diff
@@ -1995,11 +2004,11 @@ void ull_adv_done(struct node_rx_event_done *done)
 
 		rx_hdr = (void *)lll->node_rx_adv_term;
 		rx_hdr->rx_ftr.param_adv_term.status = BT_HCI_ERR_LIMIT_REACHED;
-	} else if (adv->ticks_remain_duration &&
-		   (adv->ticks_remain_duration <=
-		    HAL_TICKER_US_TO_TICKS((uint64_t)adv->interval *
-			ADV_INT_UNIT_US))) {
-		adv->ticks_remain_duration = 0;
+	} else if (adv->remain_duration_us &&
+		   (adv->remain_duration_us <=
+		    (adv->interval * ADV_INT_UNIT_US + EVENT_OVERHEAD_START_US +
+		     HAL_TICKER_TICKS_TO_US(adv->delay)))) {
+		adv->remain_duration_us = 0;
```

Comment:
as `adv->remain_duration_us = interval_us;`

```suggestion
	} else if (adv->remain_duration_us &&
		   (adv->remain_duration_us <=
		    (adv->interval * ADV_INT_UNIT_US +
		     auto_variable_for_adv_delay_of_JIT_scheduling))) {
		adv->remain_duration_us = 0;
```